cpp-package/example/mlp.cpp

```diff
@@ -144,7 +144,7 @@ void MLP() {
                                grad_req_type, aux_states);
 
   std::cout << "Training" << std::endl;
-  int max_iters = 20000;
+  int max_iters = 10;
```

Comment:
You are reducing max_iters to 10, but you did not change line 152 which has an `if (iter % 100 == 0)`
If you want to run for 10 epochs only I think you can remove the if condition on line 152.
I also suggest to rename `iter` to `epochNum` for readability.